pom.xml

```diff
@@ -357,7 +357,7 @@
         <artifactId>maven-failsafe-plugin</artifactId>
         <version>2.19.1</version>
         <executions>
-          <execution>
+<!--          <execution>
```

Comment:
any reason to keep it commented?

Reply:
no, it is my mistake, I commented this for testing quicker